packages/block-editor/src/components/editor-styles/index.js

```diff
@@ -9,15 +9,19 @@ import a11yPlugin from 'colord/plugins/a11y';
  * WordPress dependencies
  */
 import { SVG } from '@wordpress/components';
-import { useCallback, useMemo } from '@wordpress/element';
-import { useSelect } from '@wordpress/data';
+import {
+	useCallback,
+	useMemo,
+	createContext,
+	useState,
+} from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import transformStyles from '../../utils/transform-styles';
-import { store as blockEditorStore } from '../../store';
-import { unlock } from '../../lock-unlock';
+
+export const updateStyleContext = createContext();
```

Comment:
```suggestion
export const StyleOverridesContext = createContext();
```

Reply:
That is true but it seems using this style is more common. I am ok with `styleOverridesContext`too :)